app/models/lockstep/user_invite.rb

```diff
@@ -0,0 +1,11 @@
+class Lockstep::UserInvite < Lockstep::ApiRecord
```

Comment:
Do we need a separate model for this? Couldn't we had put this method under Lockstep::User? As the api call that's happening is scoped to user_accounts?

Reply:
Added [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/44/commits/2a199c1570baa68bf6e8f98d7cf272eea44e054c#)